jerry-core/profiles/README.md

```diff
@@ -120,6 +120,8 @@ defined to `1`.
   Enables or disables the [WeakMap](https://262.ecma-international.org/11.0/#sec-weakmap-objects) built-in.
 * `JERRY_BUILTIN_WEAKSET`:
   Enables or disables the [WeakSet](https://262.ecma-international.org/11.0/#sec-weakmap-objects) built-in.
+* `JERRY_BUILTIN_WEAKREF`:
+  Enables or disables the [WeakRef](https://tc39.es/proposal-weakrefs/#sec-weak-ref-objects) built-in.
```

Comment:
its better using the standard url instead draft URL， draft URL may dead in future

Reply:
Good point! Sadly i can only find documentation about the weakRef in the proposal section